app/models/course/assessment.rb

```diff
@@ -103,11 +103,8 @@ class Course::Assessment < ApplicationRecord
   # @!method self.ordered_by_date_and_title
   #   Orders the assessments by the starting date and title.
   scope :ordered_by_date_and_title, (lambda do
-    select(<<~SQL).
-      course_assessments.*, course_reference_times.start_at, course_lesson_plan_items.title
-    SQL
     joins(:lesson_plan_item).
-    merge(Course::LessonPlan::Item.ordered_by_date_and_title)
```

Comment:
Why are we removing some SQL here?

Reply:
this SQL is redundant since actually in the statement that follows (`merge(Course::LessonPlan::Item.ordered_by_date_and_title)`), we already covers `includes(reference_times: :reference_timeline)` and the `lesson_plan_items` itself is already covered inside `joins` statement

Reply:
Does this affect the amount of information we fetch? e.g. `course_lesson_plan_items.title` vs `course_lesson_plan_items.*`?

Reply:
Not at all. This scope is merely to order the assessments based on date and title (firstly by date, then by title if the date is same). As long as the information regarding the `start_at` and `title` is provided, all is safe

The reason those are deleted was because inside `Course::LessonPlan::Item.ordered_by_date_and_title`, those informations extracted by the SQL (`course_reference_times.start_at` and `course_lesson_plan_items.title`) are also extracted there